x-pack/plugins/security_solution/public/common/components/navigation/tab_navigation/tab_navigation.tsx

```diff
@@ -50,6 +51,7 @@ const TabNavigationItemComponent = ({
       href={appHref}
       onClick={handleClick}
       append={isBeta && <EuiBadge color={'#E0E5EE'}>{betaOptions?.text ?? BETA}</EuiBadge>}
+      id={tourAnchor}
```

Comment:
`TabNavigationItemComponent` already has an `id` in props. Why don't we use it here, why we're adding an additional property?

Reply:
I did this because the id we currently pass to the tabs themselves is `updates` which seems very generic and could easily be duplicated down the line. I'll switch it over but it's something to be mindful of.

Reply:
@dplumlee Can we change the `updates` id to something more specific?

Reply:
We probably could, but we also use that `id` to construct the `data-test-subj` and telemetry event ids so messing with it might not be the best thing downstream.